src/main/scala/scalaz/analytics/AnalyticsModule.scala

```diff
@@ -44,6 +44,17 @@ trait AnalyticsModule {
   implicit val doubleType: Type[Double]
   implicit val doubleNumeric: Numeric[Double]
 
+  implicit val decimalType: Type[BigDecimal]
+  implicit val decimalNumeric: Numeric[BigDecimal]
+
+  implicit val stringType: Type[String]
+  implicit val booleanType: Type[Boolean]
+  implicit val byteType: Type[Byte]
+  implicit val nullType: Type[scala.Null]
+  implicit val shortType: Type[Short]
+  implicit val timestampType: Type[java.sql.Timestamp]
```

Comment:
Hmmm I wonder whether we want to use these `java.sql` implementations.

Java best practice (as of java 8) is to use the new `java.time` package for all this tuff.
http://www.oracle.com/technetwork/articles/java/jf14-date-time-2125367.html

In this case 
`java.sql.Date` would be `java.time.LocalDate`
and
`java.sql.Timestamp` would be `java.time. LocalDateTime `

The only downside of this would be that this is java 8 onwards only.

I haven't seen anyone using anything below java 8 unless forced to in quite some time. Even enterprises seem to be on java 8+ these days. What do people think about supporting earlier java versions?

Reply:
Note that Quasar used java8 `java.time` package so that might be the way to go unless @jdegoes has strong opinions otherwise?
https://github.com/slamdata/quark/blob/master/core/src/main/scala/quasar/quark/types.scala#L22